specificatie/BRK-Bevragen/openapi.yaml

```diff
@@ -1839,12 +1851,18 @@ components:
       - "man"
       - "vrouw"
       - "onbekend"
+    TypePersoon_enum:
+      type: "string"
+      description: ":\n* `ingeschrevenpersoon`\n* `kadasterpersoon` "
+      enum:
+      - "ingeschrevenpersoon"
+      - "kadasterpersoon"
     Bedrag:
       type: "object"
       description: "Bedrag is samengesteld datatype voor het vastleggen van een hoeveelheid geld in cijfers in een bepaalde valuta."
       properties:
         som:
-          type: "string"
+          type: "integer"
```

Comment:
volgens de title kan het bedrag 2 decimalen hebben. Klopt die niet, of is het geen integer?

Als description klopt (decimalen toegestaan), dan is type: number en format: float

Reply:
Aangepast